site/source/tutorials/005_form_widgets/index.md

```diff
@@ -1,26 +1,159 @@
-- App passes handlers for onChange (update form values), onBlur (validate), and onSubmit (add worker) to WorkerForm
-	- Important to note: event handlers set on a VNode can't be reassigned (it's a Maquette restriction). So `WorkerForm.properties.onSubmit` needs to be called in a private `WorkerForm._onSubmit` function, which is then passed to `v('form')`. TextInput widgets already handle this internally, so they can have anonymous functions passed directly to their `onChange` and `onBlur` properties.
-- First and Last name are grouped in a `fieldset` with individual hidden labels for better accessibility
-- If you want to talk about styling existing widgets from `@dojo/widgets`, it would be easy to add something like `.inputWrapper` to `workerForm.css`, and add `overrideClasses: styles` to the TextInput widgets.
+# Working with forms
+
+## Overview
+This tutorial will extend on the [previous](../004_user_interactions/) tutorial where we allowed the user to interact with the application by listening for click events. In this tutorial, we will add a form to the Biz-E-Worker page so that a user can add new workers to the application. This will be done by using some of Dojo 2's built-in form widgets to allow the feature to be developed more rapidly.
+
+## Prerequisites
+You can [download](../assets/005_form_widgets-initial.zip) the demo project to get started.
+
+You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](../comingsoon.html) article.
+
+## Creating the form
+The first step to allowing the user to create new workers is to create a form. This form will contain the input elements that will accept the new worker's initial settings. To start, go to the `WorkerForm.ts` file and add this code:
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:1-4,7-9,19-24,26-28,31,33,45-48,109-111 %}
+
+This widget will render an empty form with a `submit` handler that prevents the form from being submitted to the server. Before we continue to expand on this starting point though, let's get the form integrated into the application so that we can see the form develop as we add more features.
+
+To start, add the CSS rules that the widget is going to need. Include the following rules in the `workerForm.css` file.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/styles/workerForm.css' lang:css %}
+
+Now, let's add the `WorkerForm` to the `App` class. Import the class and update the `render` method to draw it. It should be included after the `Banner` and before the `WorkerContainer` so the `render` method should look like this:
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/App.ts' lines:73,78-80,105-110 %}
+
+{% aside 'Reminder' %}
+If you can't see the application, remember to run `dojo build -w` to build the application and start the development server.
```

Comment:
I think we might be shifting to recommend `dojo build webpack -w` going forward?